### PR TITLE
Swedish Translation

### PR DESCRIPTION
**Pull request by Aresiel**
Translated everything in /mods/*/locale into Swedish. This took a while.

### CODE REVIEW

File: mods/beds/locale/beds.se.tr [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# textdomain: beds
+Fancy Bed=Fin säng
+Simple Bed=Enkel Säng
+Leave Bed=Lämna Säng
+Good morning.= God morgon.
+@1 of @2 players are in bed= @1 av @2 spelar försöker sover.
+Force night skip= Tvinga över natten
```
Comment (sfan5) [inline mods/beds/locale/beds.se.tr]: extra space here and the line above

Comment (Aresiel) [reply mods/beds/locale/beds.se.tr]: Oh, sorry

**Comment by SmallJoker** [comment]
Seems to be alright. If there are no objections it can be merged soon.
:+1:

**Comment by tacotexmex** [comment]
Swedish does not capitalize nouns like it’s done currently. ”Enkel Säng” should be ”Enkel säng” etc.

**Pull request merged**